.gitignore

```diff
@@ -1 +1,2 @@
 bower_components/
+.DS_Store
```

Comment:
@ssorallen previously suggested to me that items like this, which are not project specific, go in a user's global git ignore. I usually don't think twice about throwing things in `.gitignore`, but there might be something to be said for keeping this project-specific file strictly relevant to the project (and ignorant of the environment). What do you think?